### DISCUSSION TITLE
Int4的模型

### DISCUSSION THREAD

**Discussion by tangent2018** [OpenBMB/MiniCPM#48] [Q&A] [answered]
文档上说的MiniCPM-2B-DPO-Int4，有实现代码或者开放模型吗

**Comment by jctime** [comment]
有一个https://huggingface.co/openbmb/MiniCPM-2B-sft-int4 的版本

**Comment by zh-zheng** [comment, marked as answer]
模型在这里：https://huggingface.co/openbmb/MiniCPM-2B-dpo-int4
